### DISCUSSION TITLE
What is the current maximum allowed value for Pagination limit?

### DISCUSSION THREAD

**Discussion by michaldrabik** [trakt/trakt-api#436] [Q&A] [answered]
As title says, I am curious what is the maximum allowed items per page I can request for paginated endpoints? It is not mentioned anywhere in Pagination section in the docs. Only that the default limit is 10.

https://trakt.docs.apiary.io/#introduction/pagination

Thanks.

**Comment by tysonkerridge** [comment]
I'm pretty sure the current max is 100

**Reply by michaldrabik** [reply]
Testing out with 1000 and it's also working that's why I'm curious if there is an upper limit.

**Reply by rectifyer** [reply, marked as answer]
Depends on the method, they have a max listed if there is one. For example, `/shows/updates` has a 100 max. For calls without a specified upper limit, just use something sane like 100 or so.
